### ISSUE TITLE
Bug:  Error saving - 2

### ISSUE THREAD

**Issue by MarsRomer** [bcgov/CONN-CCBC-portal#443]

**Comment by MarsRomer** [comment]
When I clicked 'Save and continue' it wouldn't take me to the next page so I scrolled up and noticed that it said that there was an error saving
This happened a few times.  The first couple of times I think John and I were editing the same application at the same time.  I created a new application to work on and the screen captures are from the errors I encountered after that.
![image.png](https://images.zenhubusercontent.com/62686bda7385b9a511da26e2/6d7a0861-514d-4042-bc34-08277e454434)
![image.png](https://images.zenhubusercontent.com/62686bda7385b9a511da26e2/388af047-59cb-49de-8808-4348d91996ba)
